corehq/apps/app_manager/models.py

```diff
@@ -4443,6 +4455,17 @@ def set_form_versions(self, previous_version):
     def set_media_versions(self, previous_version):
         pass
 
+    def update_mm_map(self):
```

Comment:
This function is added to the creation of new versions from the release. I have some concerns about its performance implications as it runs through all forms finding multimedia references in each language. Any suggestions for better performance are welcomed.